Assets/Scripts/Utility/WeaponBasics.cs

```diff
@@ -202,6 +203,130 @@ public static string GetMagicAnimFilename(ElementTypes elementType)
             }
         }
 
+        public static string GetModdedWeaponFilename(DaggerfallUnityItem weapon)
```

Comment:
I was trying to find a way to use the Table system to break this into external data for you, but it's a bit of a pain to manage that way. The switch statement is easier and for now fits in with how DFU already does this work.

I'm OK with this handling in WeaponBasics. :)

Reply:
Great, I know it's not the cleanest way of doing stuff, but it seems like an alright option especially for the vanilla weapon types at least. Sucks that it takes up a bunch of lines, but there are worse examples of this already I suppose, lol.

Reply:
Yeah, it sort of plays on my mind how I'd like to do this better. But we're kind of stuck with interfacing to classic data and there's only so much work we can reasonably do as unpaid devs. It's fine. :)

Reply:
> Also a note that there is at least a few examples in "FormulaHelper.cs" that is checking artifacts based on their literal item-name

Right you are. :) For my part, I'm gradually (slowly) working back through all the use of string literals and trying to migrate these over to support localization in some way. This has been a real agony for me and I often feel like I'm barely making progress. What I'm trying to do (if possible) is not introduce any more cases of this in new code. That way I only have to fix the old code and not keep chasing new instances being added by new code. The gap here of course is that handling for this hasn't been added yet so there's no good solution for you to move to either.

I'm sorry if that's adding complexity for you. I can definitely make an exception in this case if we can't find another way to identify cleanly. What's one more. :)